includes/Core/Conversion_Tracking/Conversion_Events_Provider.php

```diff
@@ -35,12 +60,16 @@ public function is_active();
 	 *
 	 * @return array List of event names.
 	 */
-	public function get_event_names();
+	public function get_event_names() {
+		return array();
+	}
```

Comment:
Shouldn't it be an abstract method? Providers must implement this, mustn't they?